File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver import ActionChains
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("https://v1.training-support.net/selenium/input-events")
    print("Title of the page: "+driver.title)

    action=ActionChains(driver)

    #left click
    action.click().perform()
    frontText = driver.find_element(By.CLASS_NAME,"active").text
    print("left click: "+frontText)

    #double click
    action.double_click().perform()
    frontText = driver.find_element(By.CLASS_NAME,"active").text
    print("double click: "+frontText)

    #right click
    action.context_click().perform()
    frontText = driver.find_element(By.CLASS_NAME,"active").text
    print("right click: "+frontText)
    driver.close()

File: Selenium/Activities/Activity5.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity5 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/input-events");
        System.out.println("Title: "+driver.getTitle());
        Actions action=new Actions(driver);
        //left click
        action.click().build().perform();
        String frontText = driver.findElement(By.className("active")).getText();
        System.out.println("left click: "+frontText);
        //double click
        action.doubleClick().build().perform();
        frontText = driver.findElement(By.className("active")).getText();
        System.out.println("double click: "+frontText);
        //right click
        action.contextClick().build().perform();
        frontText = driver.findElement(By.className("active")).getText();
        System.out.println("right click: "+frontText);
        driver.close();






    }

    }
